src/util/mem.cpp

```diff
@@ -29,7 +29,7 @@ std::unique_ptr<MappedData> MappedData::from_file(const char *filename) {
   }
 
   void *mmapped_data =
-      mmap(nullptr, file_size, PROT_READ, MAP_PRIVATE, file_fd, 0);
+    mmap(nullptr, file_size, PROT_READ, MAP_SHARED, file_fd, 0);
```

Comment:
Is there any benefit to this? We aren't using this mapped region to share data among threads?

Reply:
the man page doesn't completely agree with [this SO post](https://stackoverflow.com/questions/7222164/mmap-an-entire-large-file), but I did some experimenting and it appears to be right (at least on WSL)

Reply:
Hm, fair enough. Can't hurt.